packages/react/src/schema/ReactBlockSpec.tsx

```diff
@@ -117,11 +117,74 @@ export function BlockContentWrapper<
   );
 }
 
+/**
```

Comment:
specific reason for adding this? doesn't it add a lot of extra complexity?

Reply:
Allows inference of the shape of the config based on the options provided. So it is useful complexity